packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -108,10 +114,23 @@ const FolderTreeItem = ({
     ? <IconCaretDown height="8px" width="8px" style={{ verticalAlign: 'baseline' }} /> // eslint-disable-line react/forbid-component-props
     : <IconCaretRight height="8px" width="8px" style={{ verticalAlign: 'baseline' }} />; // eslint-disable-line react/forbid-component-props
 
+  const radioButton = isSelectable ? (
+    <input
+      type="radio"
+      checked={isSelected}
+      onChange={onSelect}
+      aria-hidden // Hiding the radio button from assistive technology since they cannot be grouped correctly
+      tabIndex={-1} // Prevent tabbing to the button since it should not be read or acknowledged by assistive technology
+      className={cx('radio')}
+    />
+  ) : (
+    <div className={cx('radio-container')} />
```

Comment:
A div is a block element.  Do you want an actual div?  Can we not just add the necessary padding or margin to the sibling element?